code/fusion_fuc/include/stitcher.h

```diff
@@ -0,0 +1,17 @@
+#ifndef STITCHER_H
+#define STITCHER_H
+
+#include <opencv2/core.hpp>
+#include <string>
+
+#ifdef _WIN32
+    #define EXPORT __declspec(dllexport)
+#else
+    #define EXPORT
+#endif
+
+extern "C" {
+    EXPORT bool stitchImages(const char* img1_path, const char* img2_path, const char* output_path);
```

Comment:
this interface is not match our desgin, please refer to 
```cpp
   /**
     * Fuses two AVFrames into a single fused frame.
     *
     * This function takes two input AVFrames, frame1 and frame2, and fuses them into a single output frame, frame_fused.
     * The fusion process is performed by averaging the pixel values of the two input frames.
     *
     * @param frame1 The first input AVFrame.
     * @param frame2 The second input AVFrame.
     * @param frame_fused The output AVFrame that will contain the fused image.
     * @return True if the fusion process is successful, false otherwise.
     */
    bool image_fusion(AVFrame *frame1, AVFrame *frame2, AVFrame *frame_fused)
```

Reply:
I got it that I would solve it in next two days.